doc/en/source/https.rst

```diff
@@ -209,29 +209,67 @@ Terminology
 
 
 
-.. figure:: figures/security_architecture.png
+.. figure:: figures/Security_Architecture.png
 
 
+Note: Red markers represent IoT Device interaction with cloud and  green markers represent IoT Device interaction with Edge Node
 
-Note: In above diagram n ( n is numeric) is used for cloud node and n’ for edge node.
+ 
+
+**Flow of Requests as shown in diagram:**
+
+**Step 1** : User will make a request to IDM using his credentials to generate access token specific for that user. For example:
+
+
+.. code-block:: console
+        
+
+	curl --include \
+        --request POST \
+        --header "Content-Type: application/json" \
+        --data-binary "{
+     	\"name\": \"admin@test.com\",
+     	\"password\": \"1234\"
+   	}" \
+   	"http://KEYROCK_IP_ADDRESS:3000/v1/auth/tokens" 
+
+ 
+
+**Step 2** : IDM will return an access token to user, based on his credentials in rquest made in previous step.
 
+**Step 3** : User shares his access token ( i.e. User Access Token ) to IoT Device.
 
-FogFlow cloud node flow:
+**Step 4** : IoT Device is regstered with IDM using User Access Token as shown in `below`_ topics. As an output, IDM provides an Id and password to IoT Device based on the registration request.
```

Comment:
Then IOT devices get registered using the user access token and  in response it get device credentials(ID and password) as shown in `below`_ topics

Reply:
Comment Incorporated.